python/lsst/sconsUtils/dependencies.py

```diff
@@ -334,6 +348,72 @@ def configure(self, conf, packages, check=False, build=True):
                     return False
         return True
 
+    @staticmethod
+    def _get_config_var(name):
+        """The relevant Python is not guaranteed to be the Python
+        that we are using to run SCons so we must shell out to the
+        PATH python."""
+        pycmd = 'import distutils.sysconfig as s; print(s.get_config_var("{}"))'.format(name)
+        result = subprocess.check_output(["python", "-c", pycmd]).decode().strip()
+        # Be consistent with native interface
+        if result == "None":
+            result = None
+        return result
+
+    def configurePython(self, conf, packages, check=False, build=True):
```

Comment:
Needs a docstring. There are essentially no comments in the entire method helping people out.